check_stability.py

```diff
@@ -283,6 +286,17 @@ def wptrunner_args(self, root):
             "test_types": ["testharness", "reftest"]
         }
 
+    def extra_setup(self):
+        # https://bugs.chromium.org/p/chromium/issues/detail?id=713947
+        logger.debug("DBUS_SESSION_BUS_ADDRESS %s" % os.environ.get("DBUS_SESSION_BUS_ADDRESS"))
+        if "DBUS_SESSION_BUS_ADDRESS" not in os.environ:
+            if find_executable("dbus-launch"):
+                logger.debug("Attempting to start dbus")
+                logger.debug(subprocess.check_output(["dbus-launch"]))
+            else:
+                logger.debug("Setting DBUS_SESSION_BUS_ADDRESS to a placeholder value")
+                os.environ["DBUS_SESSION_BUS_ADDRESS"] = "/dev/null"
```

Comment:
Since we believe that this works and `dbus-launch` also seems to work in the travis environment can we land this whilst you do that research? As I say at some point I will move this into wptrunner and we can be more careful then.